vmdb/app/controllers/api_controller/conditions.rb

```diff
@@ -0,0 +1,12 @@
+class ApiController
+  module Conditions
+    #
+    # Conditions Subcollection Supporting Methods
+    #
+
+    def conditions_query_resource(object)
+      return {} unless object && object.respond_to?(:conditions)
```

Comment:
Can't this be simplified to `return {} unless object.respond_to?(:conditions)` because if `object` is `nil`, it will not `respond_to?(:conditions)`.